amy/workshops/migrations/0237_auto_20210331_1851.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.18 on 2021-03-31 18:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workshops', '0236_organization_coordinates'),
```

Comment:
The migration should depend on '0239' and be the next number - 0240

Reply:
Thanks. Is it ok to edit the dependencies and the file name manually?

Reply:
I'm not sure the tests would accept that if the `0239` migration was not found.

But you can do this:

```
git checkout develop
git pull
git checkout -
git merge develop
```

After this you can rename the migration as needed (or even drop `amy/workshops/migrations/0237_auto_20210331_1851.py` and recreate with `python manage.py makemigrations`).

Reply:
@maneesha I'm not sure you're familiarity so forgive me if this is too basic. I just wanted to provide some color in case it was needed.

Your pull request is outdated relative to the migration code, so when you used `python manage.py makemigrations` the migration was dependent on the latest migration you had available. But that's not the most current migration in the codebase now, so you code is out of date. It's nice to have the latest code when you're creating migrations so you can ensure all the changes up to this point work well together.

So in this case, I'd recommend doing the following:

Like Piotr mentioned you'll want to pull the most recent version of develop and combine your changes with git. 

After that you will want to do a couple of things:
1. Roll back your migrations `python manage.py migrate workshops zero`
2. Delete the `amy/workshops/migrations/0237_auto_20210331_1851.py` file
3. Rerun `python manage.py makemigrations`. This will create a new file with your changes.
4. Run `make database` and `make serve` so you can see that everything runs ok locally and your changes are there.
5. Make another commit with your new changes and push.